docs/openhabian.md

```diff
@@ -213,18 +207,24 @@ All password can be changed from openHABian menu.
+## Availability and Backup
+openHAB is designed to reliably run 24 hours a day, seven days a week - and so should be your server.
+This is the right time to prepare your system for disasters such as getting hit by the SD card wearout/corruption problem which is quite common on single board computers such as Raspberry Pis. openHABian has a number of features built in to enhance resilience:
 
-There are four measures and methods in openHABian to cover this matter today, but they all need some research and readup to be successfully used. *(Hopefully we will get to develop a more streamlined method to better integrate with openHAB itself - stay tuned.)*
+1. the ZRAM feature will move write intensive files (cache, logs, persistence data) into RAM to mitigate the risk of SD card corruption. See [here](https://community.openhab.org/t/zram-status/80996) for more up to date information.
+WARNING: power failure will result in some data to get lost (albeit the system should continue to run). Get an UPS. [Menu option: 38]
+2. Move the root filesystem to USB-attached memory. WARNING: USB sticks are as susceptible to flash wearout as SD cards are, making ZRAM the better choice for a standard Pi to run off its internal SD card. But you can use this option to migrate your system to a safe medium such as an SSD or HDD. [Menu option:37]
+3. Use the openHAB integrated [backup tool](https://community.openhab.org/t/recommended-way-to-backup-restore-oh2-configurations-and-things/7193/82) to interactively backup/restore your openHAB **config**.
+4. Use [Amanda Network Backup](http://www.amanda.org/) for full system backups, documentation [here](https://github.com/openhab/openhabian/blob/master/docs/openhabian-amanda.md). [Menu option: 51]
 
-1. Use openHAB integrated [backup tool](https://community.openhab.org/t/recommended-way-to-backup-restore-oh2-configurations-and-things/7193/82?u=elias_gabrielsson).
-2. Move the root filesystem to an external USB-memory. Warning: USB stick are as-susceptible to flash wearout as SD cards. [Menu option:37]
-3. (BETA) Reduce wear on SD card by moving write intensive actions temporary to RAM during operation (logs,persistant-data). Warning: power failure will result in lost data. [Menu option: 6A]
-4. (Advanced) Use [Amanda Network Backup](http://www.amanda.org/) for full system backup, longer introduction [here](https://github.com/openhab/openhabian/blob/master/docs/openhabian-amanda.md). [Menu option: 51]
+Standard openHABian install enables ZRAM (#1) by default. Once installed, you can disable ZRAM and choose to install to a safe external medium such as an SSD (#2) instead via menu options in menu 30.
+Delayed rules load (#3) will also be enabled by default. You can toggle to use this feature in menu option 44.
```

Comment:
this seems no longer the case as you removed it yesterday

Reply:
no I re-added delayed_rules to openhab2_setup()